### ISSUE TITLE
Does not work on Fedora 35

### ISSUE THREAD

**Issue by emanuc** [teejee2008/timeshift#825]
With the upgrade from Fedora 34 to 35, Timeshift no longer starts, it doesn't even work from CLI.

The logs are attached on the bug report I created for Fedora: https://bugzilla.redhat.com/show_bug.cgi?id=2019016

**Comment by gvisoc** [comment]
For RSYNC snapshots, it stops finding the storage devices and says it's operating in **Live USB mode (Restore Only)**, not finding any of my previous (Fedora 35) snapshots.

After looking the bugzilla filled by @emanuc, seems that the problem is that timeshift can't find the partitions as well. 

In this case, the error may be the same as described in [this article](https://forum.endeavouros.com/t/help-timeshift-in-live-usb-mode-as-device-uuid-not-found/14914/5).

TL;DR: the `lsblk` output has changed its format, and hence has broken this and probably other applications.

**Comment by gvisoc** [comment]
I think this is also referenced in other issues both here and in RedHat's bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2010381

**Comment by xelser** [comment]
Can someone teach me how to compile from the source? Someone fixed this problem by compiling the v21.09.1. On Fedora, timeshift is at version 20.03.4

**Comment by wmutschl** [comment]
I have the instructions in the README on my fork:
https://github.com/wmutschl/timeshift

**Comment by xelser** [comment]
Thank you very much. It's fixed!

**Comment by SafwanElmadani** [comment]
Same issue here!
![image](https://user-images.githubusercontent.com/35984199/140170270-21970134-9255-4cf2-a639-925bc38e1cad.png)

**Comment by emanuc** [comment]
Fixed momentarily thanks to a user who packaged Fedora 35 on coprs with the latest patches.

https://copr.fedorainfracloud.org/coprs/oprizal/timeshift-upstream/

**Comment by ShrirajHegde** [comment]
> Fixed momentarily thanks to a user who packaged Fedora 35 on coprs with the latest patches.
> 
> https://copr.fedorainfracloud.org/coprs/oprizal/timeshift-upstream/

@emanuc  Can you edit your issue and put this to the top and maybe edit the title to [Temp fix available]? That will make it easy for people who upgrade and come here looking for the fix.

**Comment by PolyPaul** [comment]
SImilar issue here. timeshift only launches to restore mode.
![image](https://user-images.githubusercontent.com/8498668/140774425-87deedc5-71b3-41c8-8d69-32b9e177149c.png)
```

[paul@my-laptop ~]$ timeshift-launcher
localuser:root being added to access control list
E: ts: Failed to get partition list.
App config loaded: /etc/timeshift.json

** (timeshift-gtk:57759): WARNING **: 10:55:35.077: AT-SPI: Could not obtain desktop path or name


** (timeshift-gtk:57759): WARNING **: 10:55:35.078: atk-bridge: GetRegisteredEvents returned message with unknown signature

** (timeshift-gtk:57759): WARNING **: 10:55:35.078: atk-bridge: get_device_events_reply: unknown signature

** (timeshift-gtk:57759): WARNING **: 10:55:35.078: atk-bridge: get_device_events_reply: unknown signature

** (timeshift-gtk:57759): CRITICAL **: 10:55:35.167: gee_abstract_collection_get_size: assertion 'self != NULL' failed
App config saved: /etc/timeshift.json
localuser:root being removed from access control list
access control enabled, only authorized clients can connect
SI:localuser:paul

```

**Comment by ShrirajHegde** [comment]
> SImilar issue here. timeshift only launches to restore mode.
@PolyPaul 
Take some time to read other replies

https://copr.fedorainfracloud.org/coprs/oprizal/timeshift-upstream/

**Comment by PolyPaul** [comment]
> > SImilar issue here. timeshift only launches to restore mode.
> @PolyPaul 
> 
> Take some time to read other replies
> 
> https://copr.fedorainfracloud.org/coprs/oprizal/timeshift-upstream/

Yes I have. Currently using copr already. Just thought the additional info would be helpfull. No worries.

**Comment by logicito** [comment]
Great fix, thank you

**Comment by KennthStailey** [comment]
I took a look at the spec file changes from the `timeshift-20.03-4.fc35.x86_64` and the copr. They are very minor, just a new version number, remove an old patch and change the location of the config file. LGTM
```
--- timeshift-20.03-4.spec      2021-07-23 19:19:10.000000000 -0400
+++ timeshift-21.09.1.spec      2021-11-15 11:47:15.948683292 -0500
@@ -7,8 +7,8 @@
 
 
 Name:           timeshift
-Version:        20.03
-Release:        4%{?dist}
+Version:        21.09.1
+Release:        1%{?dist}
 Summary:        System restore tool for Linux
 
 License:        GPLv3+ or LGPLv3+
@@ -16,7 +16,7 @@
 Source0:        %{url}/archive/v%{version}/%{name}-%{version}.tar.gz
 
 # https://github.com/teejee2008/timeshift/pull/566
-Patch0:         timeshift-20.03-timeshift-pt_BR.po-Fix-newlines.patch
+#Patch0:         timeshift-20.03-timeshift-pt_BR.po-Fix-newlines.patch
 
 BuildRequires: make
 BuildRequires:  desktop-file-utils
@@ -122,8 +122,7 @@
 %ghost %attr(644, root, root) %{_sysconfdir}/cron.d/timeshift-boot
 %ghost %attr(644, root, root) %{_sysconfdir}/cron.d/timeshift-hourly
 %ghost %attr(664, root, root) %{_sysconfdir}/timeshift.json
-%config %{_sysconfdir}/default/timeshift.json
-
+%config %{_sysconfdir}/timeshift/default.json
 
 
 %changelog
```
